tests/config.yml

```diff
@@ -37,4 +37,3 @@ modules:
   # These settings only apply to modules/module_utils.
   # It is not possible to declare supported Python versions for controller-only code.
   # All Python versions supported by the controller must be supported by controller-only code.
-  
```

Comment:
Seems an unintentional removal?

Reply:
That was to pass the new `ansible-lint` tests:

```
WARNING  Listing 112 violation(s) that are fatal
.
.
.
yaml[new-line-at-end-of-file]: No new line character at the end of file
tests/config.yml:40
.
.
.
                       Rule Violation Summary                        
 count tag                           profile    rule associated tags 
     3 yaml[comments]                basic      formatting, yaml     
     2 yaml[indentation]             basic      formatting, yaml     
     2 yaml[new-line-at-end-of-file] basic      formatting, yaml     
     3 yaml[trailing-spaces]         basic      formatting, yaml     
     1 yaml[truthy]                  basic      formatting, yaml     
    99 sanity[cannot-ignore]         production idiom                
     2 fqcn[deep]                    production formatting (warning) 
```